Python/bytecodes.c

```diff
@@ -2216,9 +2221,8 @@ dummy_func(
             assert(index < FT_ATOMIC_LOAD_SSIZE_RELAXED(mod_keys->dk_nentries));
             PyDictUnicodeEntry *ep = DK_UNICODE_ENTRIES(mod_keys) + index;
             PyObject *attr_o = FT_ATOMIC_LOAD_PTR_RELAXED(ep->me_value);
-            DEAD(mod_keys);
             // Clear mod_keys from stack in case we need to deopt
-            POP_DEAD_INPUTS();
+            POP_INPUT();
```

Comment:
I think these macros are a bit easier to read when they include the name of the relevant argument, e.g., `POP_INPUT(mod_keys)`, and the code generator ensures that the name matches the top of stack.